AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1036,6 +1037,21 @@ - (void)__setNeedsLayout
   }
 }
 
+// If not rasterized (and therefore we certainly have a view or layer),
```

Comment:
This comment still seems relevant, but I believe it was originally in the bridge method… The one without two underscores.  It does not seem to be referencing the actions taken in the method here, even though it mentions schedule display that is intended to be from the position of -setNeedsDisplay calling this method and having some internal knowledge about the fact that it does that.